evergreen/config.yml

```diff
@@ -819,7 +867,23 @@ tasks:
       test_filter: SyncTests
       test_executable_name: "realm-sync-tests"
 
-- name: object-store-tests-local-baas
+# These are local object store tests; baas is not started, however some use the sync server
+- name: object-store-tests
+  tags: [ "test_suite_local_baas", "test_suite_remote_baas", "for_pull_requests" ]
+  exec_timeout_secs: 3600
+  commands:
+  - func: "compile"
+    vars:
+      target_to_build: ObjectStoreTests
+  - func: "run tests"
+    vars:
+      test_label: objstore-local
+      test_executable_name: "realm-object-store-tests"
+      verbose_test_output: true
+  - func: "check branch state"
+
+# These are baas object store tests that run against baas running locally
+- name: local-baas-integration-tests
```

Comment:
why do we need this? aren't all tests running against remote baas?

Reply:
Right - they aren't used anymore. These were just kept around in case we wanted to switch some of the tasks back to use a local baas, but I don't see any benefit to keeping them. I'll remove them.